src/main/java/carpet/utils/FabricAPIHooks.java

```diff
@@ -0,0 +1,44 @@
+package carpet.utils;
+
+import carpet.network.CarpetClient;
+import com.mojang.blaze3d.systems.RenderSystem;
+import net.fabricmc.fabric.api.client.rendering.v1.WorldRenderEvents;
+import net.fabricmc.loader.api.FabricLoader;
+import net.fabricmc.loader.api.SemanticVersion;
+import net.fabricmc.loader.api.Version;
+import net.fabricmc.loader.api.VersionParsingException;
+
+public class FabricAPIHooks {
+
+    public static final boolean WORLD_RENDER_EVENTS = hasMod("fabric-rendering-v1", "1.5.0");
```

Comment:
technical question - does it need to be bound / harcoded to that specific version, or is works as >= ?

Reply:
all good, see below